data/fields/archaeological_site.json

```diff
@@ -1,5 +1,22 @@
 {
     "key": "archaeological_site",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "baths": "Baths",
+            "city": "Historic City",
+            "crannog": "Crannóg",
```

Comment:
I think this is spelled just _Crannog_ in English, isn't it? See e.g. https://en.wikipedia.org/wiki/Crannog